index.html

```diff
@@ -1,24 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Portfolio Microverse!</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/port.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Mohamed Portfolio</title>
 </head>
 <body>
-    <header>
-        <div>
-            <img src="src/img/logo.png" class="logo" alt="Portfolio">
-            <img src="src/img/menu-button.png" class="menu-button" alt="Menu">
-        </div>
+  <!--header-->
+ <div class="bk_container">
```

Comment:
- [ ] You should apply the background image on the `headline` section and it doesn't include the `toolbar/header` section.

- [ ] You should use semantically meaningful sections. For example, 
```
<body>
   <header>
       logo
       Navbar
   </header>
   <section class='headline'>
    <div class='container' >
          ......
     </div>
   </section>
</body>
```
If you have to use a container for the purpose of apply padding, margins, and background images, you can make it within the semantically meaningful section elements such `<header>, <section>` as in above outline.